kyo-prelude/shared/src/main/scala/kyo2/Var.scala

```diff
@@ -26,22 +26,27 @@ object Var:
     inline def update[V](inline f: V => V)(using inline tag: Tag[Var[V]]): Unit < Var[V] =
         Effect.suspend[Unit](tag, (v => f(v)): Update[V])
 
-    final class RunOps[V](dummy: Unit) extends AnyVal:
-        def apply[A, S](state: V)(v: A < (Var[V] & S))(using tag: Tag[Var[V]], frame: Frame): A < S =
-            Effect.handle.state(tag, state, v) {
-                [C] =>
-                    (input, state, cont) =>
-                        input match
-                            case input: Get[?] =>
-                                (state, cont(state))
-                            case input: Set[?] =>
-                                (input(), cont(()))
-                            case input: Update[?] =>
-                                (input(state), cont(()))
-            }
-    end RunOps
-
-    inline def run[V >: Nothing]: RunOps[V] = RunOps(())
+    private inline def runWith[V, A, S, B, S2](state: V)(v: A < (Var[V] & S))(
```

Comment:
I had initially left this method `public` but it doesn't seem very useful, so I made it `private` to reduce the API surface.